tests/ui/suggestions/as-ref.stderr

```diff
@@ -108,6 +106,10 @@ LL |     let y: Result<&usize, usize> = x;
    |
    = note:   expected enum `Result<&usize, usize>`
            found reference `&Result<usize, usize>`
+help: try using `.as_ref()` to convert `&Result<usize, usize>` to `Result<&usize, usize>`
+   |
+LL |     let y: Result<&usize, usize> = x.as_ref();
+   |                                     +++++++++
```

Comment:
This is wrong tho, `.as_ref()` will return `Result<&usize, &usize>`, so this is not a good suggestion. Similarly the suggestion to the previous error *was* correct.